x-pack/test/functional/apps/maps/group3/reports/index.ts

```diff
@@ -64,6 +79,8 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
       await PageObjects.reporting.clickGenerateReportButton();
 
       const percentDiff = await measurePngDifference('example_map_report');
+      // TODO: lower the threshold to 0.03. Currently this test is not passing with a low diff percentage due to
+      // unwanted behavior: https://github.com/elastic/kibana/issues/156623
```

Comment:
I did not lower the threshold for the maps embeddable example test, after noticing https://github.com/elastic/kibana/issues/156623

cc @nreese